src/graphics/opengl/OpenGL.cpp

```diff
@@ -182,9 +182,11 @@ void OpenGL::Iterate() {
 #ifdef DEBUG
 	GLenum errcode;
 	if ( ( errcode=glGetError() ) != GL_NO_ERROR ) {
+#ifdef __linux__ // gluErrorString can't be found on window
```

Comment:
Better to use #ifndef _WIN32 because there can be other POSIX systems like FreeBSD.
Actually this function may exist in MinGW too.
Maybe it's possible to check some macro from GL library, i.e. #ifdef HAS_GLU_ERROR_STRING (just example), or just check for any glu-specific macro set by GLU library?

Reply:
Fixed, it uses #ifndef VISUAL_STUDIO now